tests/org.eclipse.n4js.flowgraphs.tests/model/Expressions.n4js.xt

```diff
@@ -0,0 +1,696 @@
+	// TemplateExpression XPECT path from '"start"' to '"end"' -->
+	// TemplateExpression XPECT commonPreds of '"start"' and '"end"' --> "start"
+	// TemplateExpression XPECT succs at '"start"' --> `a ${
+	// TemplateExpression XPECT succs at 'a ${' --> "b"
+	// TemplateExpression XPECT succs at '"b"' -->  c`
+	// TemplateExpression XPECT succs at ' c`' --> `a ${"b"} c`
+	// TemplateExpression XPECT succs at '`a ${"b"} c`' --> "end"
+	"start";
+	`a ${"b"} c`;
+	"end";
+}
+
+function ThisExpression() {
+	// ThisExpression XPECT container --> FunctionDeclaration::Block
+	// ThisExpression XPECT allIslandElems -->
+	// ThisExpression XPECT allCatchBlocks -->
+	/* ThisExpression XPECT allEdges ---
+		"start" --> this,
+		this --> "end"
+	--- */
+	// ThisExpression XPECT allPaths --> "start" -> this -> "end"
+	// ThisExpression XPECT allPaths direction 'Backward' --> "end" -> this -> "start"
+	// ThisExpression XPECT path from '"start"' to '"end"' -->
+	// ThisExpression XPECT commonPreds of '"start"' and '"end"' --> "start"
+	// ThisExpression XPECT succs at '"start"' --> this
+	// ThisExpression XPECT succs at 'this' --> "end"
+	"start";
+	this;
+	"end";
+}
+
+
+function UnaryExpression() {
+	// UnaryExpression XPECT container --> FunctionDeclaration::Block
+	// UnaryExpression XPECT allIslandElems -->
+	// UnaryExpression XPECT allCatchBlocks -->
+	/* UnaryExpression XPECT allEdges ---
+	"start" --> true,
+		true --> !true,
+		!true --> "end"
+	--- */
+	// UnaryExpression XPECT allPaths --> "start" -> true -> !true -> "end"
+	// UnaryExpression XPECT allPaths direction 'Backward' --> "end" -> !true -> true -> "start"
+	// UnaryExpression XPECT path from '"start"' to '"end"' -->
+	// UnaryExpression XPECT commonPreds of '"start"' and '"end"' --> "start"
+	// UnaryExpression XPECT succs at '"start"' --> true
+	// UnaryExpression XPECT succs at 'true' --> !true
+	// UnaryExpression XPECT succs at '!true' --> "end"
+	"start";
+	!true;
+	"end";
+}
+
+function * YieldExpression(i) {
+	// YieldExpression XPECT container --> FunctionDeclaration::Block
+	// YieldExpression XPECT allIslandElems -->
+	// YieldExpression XPECT allCatchBlocks -->
+	/* YieldExpression XPECT allEdges ---
+		"start" --> i,
+		i --> 1,
+		1 --> yield 1,
+		yield 1 --> i = yield 1,
+		i = yield 1 --> "end"
+	--- */
+	// YieldExpression XPECT allPaths --> "start" -> i -> 1 -> yield 1 -> i = yield 1 -> "end"
+	// YieldExpression XPECT allPaths direction 'Backward' --> "end" -> i = yield 1 -> yield 1 -> 1 -> i -> "start"
+	// YieldExpression XPECT path from '"start"' to '"end"' -->
+	// YieldExpression XPECT commonPreds of '"start"' and '"end"' --> "start"
+	// YieldExpression XPECT succs at '"start"' --> i
+	// YieldExpression XPECT succs at 'i' --> 1
+	// YieldExpression XPECT succs at '1' --> yield 1
+	// YieldExpression XPECT succs at 'yield 1' --> i = yield 1
+	// YieldExpression XPECT succs at 'i = yield 1' --> "end"
+	"start";
+	i = yield 1;
+	"end";
+}
+
+
+
+
```

Comment:
compensating for lines missing at the end of other files?

Reply:
yay